app/assets/stylesheets/colors.css.scss

```diff
@@ -0,0 +1,10 @@
+// See http://sachagreif.com/sass-color-variables
+
+// Color definitions
+$fsek_orange: #eb7125;
+$black: #fff;
+
+// Function definitions
+$text_color: $black;
```

Comment:
Name of variable `text_color` should be written in all lowercase letters with hyphens instead of underscores